### DISCUSSION TITLE
👋 Welcome to Discussions!

### DISCUSSION THREAD

**Discussion by jpmcb** [open-sauced/app#3620] [Announcements]
Welcome to the OpenSauced app open source discussions forum! 👋🏼 

Here, you'll be able to engage with the OpenSauced core team, ask questions, and propose new ideas.

You'll also see some special core team only sections - primarily, "Proposals": this is where the core team lays out some big ideas for the app (that might not quite fit in a single issue or ticket). Feel free to join in the discussion!

**Comment by shubhamchasing** [comment]
Hey @jpmcb 👋 
I have been fixing bugs and raised issues for the past few months and would love to contribute more, can you please help me with how I can contribute to the features as most of the features are labeled `core-team`?

**Reply by jpmcb** [reply]
Hi @shubhamchasing - thanks for reaching out! And thank you so much for your contributions so far!!

Most of the `core-team` labeled work are things that require alot of coordination within our internal engineering team (for things like our close-sourced API, infrastructure, design schematics, etc. etc.). We also move relatively fast, so sometimes, it can be difficult for outside contributors tackling big tasks to stay on the same page for things we need delivered quickly.

Keep an eye out for stuff that isn't labeled as `core-team` or, if you're passionate about a certain bug that _is labeled with `core-team`_, create a detailed proposal in the issue on how you would fix it / implement it and the core engineering team can review it and assign you the issue if it seems amicable.

**Reply by shubhamchasing** [reply]
Thanks, John!
